msteams-platform/task-modules-and-cards/cards/cards-format.md

```diff
@@ -492,6 +492,219 @@ The following image shows the default view of the Adaptive Card when you have no
 
 :::image type="content" source="../../assets/images/Cards/small-width-adaptive-card.png" alt-text="Small width Adaptive Card view":::
 
+### Adaptive Card responsive layout
+
+Adaptive Cards usually match the look and feel of the host application and have a fixed layout that remains consistent across Teams mobile and desktop clients.
+
+Adaptive Card responsive layout helps you to design Adaptive Cards with different layouts that adapt to different card widths in the context where the card is displayed. When you design Adaptive Card for your app, ensure that you provide a better user experience across chat, channels, and meeting chat in Teams desktop and mobile clients.
+
+[Add image for responsive Adaptive card in mobile and desktop.]
+
+#### Design responsive Adaptive Cards
+
+You can design a responsive Adaptive Card using the `targetWidth` property, which allows you to show or hide any element based on the card's width.
+
+You can set the `targetWidth` property to any element in your Adaptive Card to create different layouts for different card widths. The following table provides the available values of the `targetWidth` property for different layouts:
```

Comment:
To design a responsive Adaptive Card, use the `targetWidth` property on any element:
- Setting `targetWidth` will cause an element to be shown or hidden depending on the card's width
- By setting different target widths on different elements, you essentially create different layouts

This table lists the available `targetWidth` values:

Reply:
updated